src/spc_django/settings.py

```diff
@@ -141,6 +141,7 @@
 EMAIL_HOST_USER = config('EMAIL_ID')
 EMAIL_HOST_PASSWORD = config('EMAIL_PASSWORD')
 EMAIL_USE_SSL = True
+TO_EMAIL = config('TO_EMAIL_ID')
```

Comment:
You should probably rename this variable, intention is not very clear
Also add comments to show what both email ID's represent in the settings and .env